EchoPro/computation/operations.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import pandas as pd
+from ..utils.monkey_patch_dataframe import patch_method_to_DataFrame
+from typing import Union , List
+from typing import Callable
+    
+@patch_method_to_DataFrame( pd.DataFrame )
+def discretize_variable( dataframe: pd.DataFrame , 
+                         bin_values: np.ndarray ,
+                         bin_variable: str ):
```

Comment:
Similar to the case with `quantize_variables`, I think `discretize_variable` may be better named as `bin_variable`